constants/queryKeys.ts

```diff
@@ -72,13 +73,20 @@ export const QUERY_KEYS = {
 			amount: string,
 			networkId: NetworkId
 		) => ['convert', '1inch', quoteCurrencyKey, baseCurrencyKey, amount, networkId],
+		approveAddress1Inch: ['convert', '1inch', 'approve', 'address'],
 	},
 	TokenLists: {
 		Synthetix: ['tokenLists', 'synthetix'],
+		OneInch: ['tokenLists', '1inch'],
 	},
 	CMC: {
 		Quotes: (currencyKeys: CurrencyKey[]) => ['cmc', 'quotes', currencyKeys.join('|')],
 	},
+	CoinGecko: {
+		CoinList: ['cg', 'coinList'],
+		TokenPrices: (tokenAddresses: string[]) => ['cg', 'prices', tokenAddresses.join('|')],
+		Prices: (priceIds: string[]) => ['cg', 'prices', priceIds.join('|')],
```

Comment:
It might make more sense in the future to make an endpoint for common priceIds... (for example that would always have ETH/SNX/sUSD price, etc)